cli/tests/unit/test_node.py

```diff
@@ -191,6 +191,87 @@ def test_node_diagnostics_download(mock_get_diagnostics_list, mock_do_request,
         stream=True)
 
 
+@mock.patch('dcos.config.get_config_val')
+@mock.patch('dcos.http.get')
+@mock.patch('dcoscli.log.dcos_log_enabled')
+def test_dcos_log_v2_leader_mesos(mocked_dcos_log_enabked, mocked_http_get,
```

Comment:
there are multiple occurences of `mocked_dcos_log_enabked` instead of `mocked_dcos_log_enabled` in this file.